pyproject.toml

```
@@ -11,6 +11,9 @@ python-dotenv = "^0.14.0"
 
 [tool.poetry.dev-dependencies]
 
+[tool.poetry.group.dev.dependencies]
+yapf = "^0.32.0"
```

When I ran `poetry install` I got a RuntimeError: `The Poetry configuration is invalid - Additional properties are not allowed ('group' was unexpected)`

This change fixed the install issue:
```suggestion
[tool.poetry.dev-dependencies]
yapf = "^0.32.0"
```

---

This is a bit weird, I think it's because I have the latest version of Poetry (1.2.0) installed which has switched to using groups for dependencies instead of a dedicated dev-dependency specification, and it's not backwards compatible. I think the correct solution might be to change the `[build-system]` table to read

```
[build-system]
requires = ["poetry-core>=1.1.0"]
build-backend = "poetry.core.masonry.api"
```

which I think will support earlier poetry versions as long as they have poetry-core installed, but I don't know enough about poetry to be sure

---

Or I just remove it entirely since it's just for my own formatting :D

---

You were right - I was using an old version of poetry. Please do not make the change I requested.